packages/plugin-core/src/components/pods/PodControls.ts

```diff
@@ -552,4 +554,23 @@ export class PodUIControls {
     });
     return podIdQuickPick?.label;
   }
+
+  /**
+   * Prompt user to select the copy as format
+   */
+  public static async promtToSelectCopyAsFormat(): Promise<
+    CopyAsFormat | undefined
```

Comment:
Is there ever going to be a case where the format is undefined?

Reply:
When user press `Esc`